numpy/core/_dtype.py

```diff
@@ -201,29 +201,29 @@ def _struct_dict_str(dtype, includealignedflag):
     # Build up a string to make the dictionary
 
     # First, the names
-    ret = "{'names':["
-    ret += ",".join(repr(name) for name in names)
+    ret = "{'names': ["
+    ret += ", ".join(repr(name) for name in names)
 
     # Second, the formats
-    ret += "], 'formats':["
-    ret += ",".join(
+    ret += "], 'formats': ["
+    ret += ", ".join(
```

Comment:
This is a mistake. If you add a space before the function, it will changes the complete functioning of variable, Remove the spacing.

Reply:
sorry, I believe we're having a misunderstanding here?

Reply:
There is a difference between joining strings.

Reply:
print(",".join("1234"))
print(", ".join("1234"))

Test both the codes, they will give different outputs. We need to add spaces where there is no change in the output.

Reply:
@shubham11941140, the whole point in this PR is that @anntzer _wants_ to change the output